StatusList.js

```diff
@@ -19,13 +19,19 @@ export default class RevocationList {
   isRevoked(index) {
     return this.bitstring.get(index);
   }
+}
+
+export default class StatusList extends RevocationList {
```

Comment:
This is backwards -- the StatusList should be the base class and, if we want a RevocationList class, it should extend the StatusList class and provide specific names for its status functions. Also, do not use `default` exports.

Reply:
I recommend trying to get away with not needing subclasses for different types of statuses -- and instead just use generic names for the methods.

Reply:
Updated here [9363d26](https://github.com/digitalbazaar/vc-status-list/pull/1/commits/9363d26883ec5695754b8f229230ed0c723c6edd)